lib/read.js

```diff
@@ -23,6 +23,12 @@ var zlib = require('zlib')
 
 module.exports = read
 
+/**
+ * @const
+ * whether current node version has brotli support
+ */
+var hasBrotliSupport = process.versions ? 'brotli' in process.versions : false
```

Comment:
It looks like the code coverage only shows one of these branches are ever run. Under what Node.js versions would the other branch run?